CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ future.
 - Fixed a bug where clustered round robin check execution executed checks
 too often.
 - Fixed a bug where users could accidentally create invalid gRPC handlers.
+- `Issued` & `History` are now set on keepalive events.
```

Comment:
You'll need to rebase on master so this entry does not go under the `2.0.0-beta.3-1` release!

Reply:
Rebased!